hack/docs/internal/cronjob-tutorial/generate_cronjob.go

```diff
@@ -507,7 +509,7 @@ var testEnv *envtest.Environment
 	/*
 		Then, we start the envtest cluster.
 	*/`)
-	CheckError("fixing suite_test.go", err)
+	CheckError("updating suite_test.go to add text to show where envtest cluster start", err)
```

Comment:
@camilamacedo86 I think it is good to add details of what is the intention when an error happens.

One thing I'm concerning is:
Logs generated by `CheckErrors` can be very long. Because the `err` from `pluginutil.InsertCode` usually presents the file content, see:
https://github.com/kubernetes-sigs/kubebuilder/blob/v3.11.1/pkg/plugin/util/util.go#L75

It is helpful since we can view to the raw content to trace the error.
Well, I'm wondering if we can make the logs `formatted`, such as adding indention before printing the raw file content?

Reply:
sure, we can add the indentation to seperate the file content and actual error.